honeybadgermpc/poly_commit_const.py

```diff
@@ -68,13 +68,13 @@ def gen_pc_const_crs(t, alpha=None, g=None, h=None, ghat=None):
     assert type(h) in (G1, nonetype)
     assert type(ghat) in (G2, nonetype)
     if alpha is None:
-        alpha = ZR.random()
+        alpha = ZR.random(0)
```

Comment:
These shouldn't be by default hard coded. The seeds should be set from the command line.